camayoc/tests/rho/test_profile.py

```diff
@@ -368,6 +370,8 @@ def test_edit_hosts(isolated_filesystem):
     rho_profile_edit.close()
     assert rho_profile_edit.exitstatus == 0
 
+    if new_hosts.endswith('0/24'):
+        new_hosts = new_hosts.replace('0/24', '\[0:255\]')
```

Comment:
If you are replacing the `'0/24'` with `'\[0:255\]'` every time, should you just go ahead and use the second?

Reply:
I am replacing that because I want to check if RHO is doing that. It should accept the hosts range in CIDR notation and convert to the Ansible notation.